<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registe</class>
 <widget class="QWidget" name="registe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>269</width>
     <height>153</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="user">
      <property name="text">
       <string>用户</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>检查</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="password">
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="user_2">
      <property name="text">
       <string>管理员密码</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="adminPassword"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="adminUser"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="password_2">
      <property name="text">
       <string>管理员用户</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="user_3"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="password_3"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>权限等级</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>管理员</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>可操作</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>仅查看</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入管理员用户和密码获得注册权限</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
